python/pytests/golden/sqrt_test/test_sqrt.jsonl

```diff
@@ -0,0 +1,3 @@
+{"_time":"2021-01-01T00:00:00.000000000","_key":"A","m":5.7,"sqrt_m":2.3874672773}
+{"_time":"2021-01-01T00:00:00.000000000","_key":"A","m":6.3,"sqrt_m":2.5099800796}
+{"_time":"2021-01-02T00:00:00.000000000","_key":"B","m":null,"sqrt_m":null}
```

Comment:
seeing lower precision results than old tests: https://kaskada.io/docs-site/kaskada/main/fenl/catalog.html#sqrt

is this due to some rounding in python?

Reply:
Could be lower precision in Python, or how the JSON represents it. I'm not sure I'd worry about it now.

Reply:
My guess is this is related to Pandas and how it prints the JSON:

https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_json.html

Note the default precision is 10 digits.